README.md

```diff
@@ -43,6 +43,19 @@ Data API builder is Open Source and works on any platform. It can be executed on
 
 To get started quickly with Data API builder for Azure Databases, you can use the [Getting Started](https://learn.microsoft.com/azure/data-api-builder/get-started/get-started-with-data-api-builder) tutorial, that will help to get familiar with some basic tools and concepts while giving you a good experience on how much Data API builder for Azure Databases can make you more efficient, but removing the need to write a lot of plumbing code.
 
+### Walk-through video
+
+<p>
+  <a href="https://www.youtube.com/watch?v=xAlaoDQolLw" target="_blank">
+    <img src="https://img.youtube.com/vi/xAlaoDQolLw/0.jpg" alt="Play Video" width="280" />
+  </a><br/>
+  Demo source code: https://aka.ms/dab/startrek
+</p>
+
+#### Install Data API builder's cross-platform  command line
+    dotnet tool install microsoft.dataapibuilder --global
```

Comment:
1. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code
1. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

https://github.com/github-linguist/linguist/blob/f0aebbe90d3b9b9bae5b7258f99370def1ee489f/lib/linguist/languages.yml#L6695-L6703

Reply:
I do not understand. 4 spaces is a code block.

Reply:
ah, gotcha. Seems GitHubs markdown docs are the only ones who don't mention that. These docs do: https://www.jetbrains.com/help/hub/markdown-syntax.html#indented-code-blocks